src/xdocs/config_filters.xml

```diff
@@ -2232,6 +2232,250 @@ public class UserService {
+        </source>
+        <p>
+          To configure the filter to suppress audit events only on a check whose id is
+          <code>ignoreMe</code>:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+    &lt;property name=&quot;idPattern&quot; value=&quot;ignoreMe&quot;/&gt;
+  &lt;/module&gt;
+  &lt;module name=&quot;LineLength&quot;&gt;
+    &lt;property name=&quot;max&quot; value=&quot;15&quot;/&gt;
+  &lt;/module&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;MagicNumber&quot;&gt;
+      &lt;property name=&quot;id&quot; value=&quot;ignoreMe&quot;/&gt;
+    &lt;/module&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+int a = 42; // ok
+int longVarName = 2; // LineLength violation
+        </source>
+        <p>
+          To configure the filter to suppress audit events for the current and next 2 lines:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+    &lt;property name=&quot;lineRange&quot; value=&quot;2&quot;/&gt;
+  &lt;/module&gt;
+  &lt;module name=&quot;UniqueProperties&quot;/&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+key.one=41 # SUPPRESS CHECKSTYLE the reason for this is
+key.one=42 # ok
+key.one=43 # ok
+key.one=44 # violation
+        </source>
+        <p>
+          To configure the filter to suppress audit events for the current and previous line:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+    &lt;property name=&quot;lineRange&quot; value=&quot;-1&quot;/&gt;
+  &lt;/module&gt;
+  &lt;module name=&quot;UniqueProperties&quot;/&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+key.one=41 # violation
+key.one=42 # ok
+key.one=43 # SUPPRESS CHECKSTYLE the reason for this is
+key.one=44 # violation
+        </source>
+        <p>
+          To configure the filter to accept variable <code>checkPattern</code>
+          and <code>lineRange</code>:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+    &lt;property name=&quot;nearbyTextPattern&quot;
+      value=&quot;SUPPRESS CHECKSTYLE (\w+) - ([+-]\d+) Lines&quot;/&gt;
+    &lt;property name=&quot;nearbyTextPattern&quot; value=&quot;$1&quot;/&gt;
+    &lt;property name=&quot;lineRange&quot; value=&quot;$2&quot;/&gt;
+  &lt;/module&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;MagicNumber&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+int a = 1; // SUPPRESS CHECKSTYLE MagicNumber - +3 Lines
```

Comment:
`@cs-: MagicNumber for +3 lines`
to make it more easy to read. 
and show example of custom `nearbyTextPattern` that we suggest as compact approach.

Reply:
Done.

Reply:
Not done, raised again.

Reply:
Done now. My bad, I didn't look closely.